### ISSUE TITLE
Need a way to add circuit drawing styles without loosing the default ones

### ISSUE THREAD

**Issue by nonhermitian** [Qiskit/qiskit-terra#8991]
### What should we add?

Currently to add circuit drawing styles one needs a setup file that sets the default path to the styles, eg something that looks like this:

```
[default]
circuit_mpl_style_path = ~/.qiskit
```

However when doing so one looses access to the builtin options, save for default.  It would be nice if the `settings.conf` file appended styles in a dir to a set that includes the default rather than changing the path.  That way an user has access to both builtin and custom.

**Comment by mtreinish** [comment]
This is a bug, the intent of that was it was an additive path as you're describing. If it's excluding the defaults (assuming there isn't a name collision) then that's a bug in the config file handling.

**Comment by DanielAja** [comment]
I'd like to work on this

**Comment by nonhermitian** [comment]
Hmm, let me try again, could be my mistake

**Comment by nonhermitian** [comment]
If I do `qc.draw('mpl', style='iqx')` then I get


/Users/paul/mambaforge/envs/qiskit/lib/python3.10/site-packages/qiskit/visualization/circuit/matplotlib.py:163: UserWarning: Style JSON file 'iqx.json' not found in any of these locations: /Users/paul/mambaforge/envs/qiskit/lib/python3.10/site-packages/qiskit/visualization/circuit/styles/iqx.json, ~/.qiskit/iqx.json, iqx.json. Will use default style.
  self._style, def_font_ratio = load_style(style)

which I guess I assumed was due to messing around with the paths, but it looks like the JSON files for the default styles are not found on any of my machines.

**Comment by enavarro51** [comment]
This might have something to do with #8306. The `styles` dir was moved to `visualization/circuit`. You might still have a `styles` dir under `visualization`, but not sure why they would not have been moved.

**Comment by mtreinish** [comment]
If you moved the directory the json files lived in you'll need to update the manifest file: https://github.com/Qiskit/qiskit-terra/blob/main/MANIFEST.in#L9 That file controls non-python data files we include in the package.

**Comment by enavarro51** [comment]
@mtreinish Yep, that's it. Second time that file has caught me. Do you want me to do a PR to update the manifest file?
